RELEASE.md

```diff
@@ -76,7 +76,10 @@ Maintaining the release branches for older minor releases happens on a best effo
 
 A few days before a major or minor release, consider updating the dependencies.
 
-Then create a pull request against the main branch.
+Note that we use [Dependabot](.github/dependabot.yml) to automatically update most things.
+See the [dependencies GitHub label](https://github.com/prometheus/prometheus/labels/dependencies) for a list of pending updates.
```

Comment:
Perhaps express somehow that this is a continuous process, so that when it comes to a release, most work is already done.

As is, it could be understood that the release shepherd has to push a button to initiate the automatic update or something.

Suggestion:

Note that we use [Dependabot](.github/dependabot.yml) to continuously update most things automatically. Therefore, mest dependencies should be up to date already. Check the [dependencies GitHub label](https://github.com/prometheus/prometheus/labels/dependencies) to see if there are any pending updates.

Reply:
Yes, that's better. Done.